lib/livebook/session.ex

```diff
@@ -1709,6 +1714,9 @@ defmodule Livebook.Session do
   #     to reflect the new `Livebook.Session.Data`
   #
   defp handle_operation(state, operation) do
+    # I am under hte assumption that in some place inside
+    # this method the operations get sent to session_live.ex,
+    # is this correct? If so, which code is in charge of doing so?
```

Comment:
a little lost here. It looks like `session_live` does get the `:add_cell_evaluation_output` operation from this function. Where does that happen?